lib/parking_lpr_web/controllers/api_controller.ex

```diff
@@ -0,0 +1,40 @@
+defmodule ParkingLprWeb.ApiController do
+  use ParkingLprWeb, :controller
+  alias ParkingLpr.{Event}
+  alias ParkingLprWeb.Router.Helpers, as: Routes
+
+  def index(conn, _params) do
+    json(conn, %{
+      count: Event.count_events(),
+      events: Event.list_event_ids()
+    })
```

Comment:
```suggestion
    event_ids = Event.list_event_ids()
    json(conn, %{
      count: length(event_ids),
      events: event_ids
    })
```

Getting the count from the DB is great for when it saves you from having to retrieve all the records. In this case though you are already pulling all the IDs, so something like the above would let you just count them in code and save a second (granted, tiny) DB call.